daemon/main.go

```diff
@@ -117,28 +117,29 @@ func checkMinRequirements() {
 	// Checking for bpf_features
 	globalsDir := filepath.Join(config.RunDir, "globals")
 	if err := os.MkdirAll(globalsDir, defaults.RuntimePathRights); err != nil {
-		log.Fatalf("Could not create runtime directory %s: %s", globalsDir, err)
+		log.Fatalf("Could not create runtime directory %q: %s", globalsDir, err)
 	}
 	if err := os.Chdir(config.RunDir); err != nil {
-		log.Fatalf("Could not change to runtime directory %s: \"%s\"",
+		log.Fatalf("Could not change to runtime directory %s: %s",
 			config.RunDir, err)
 	}
-	_, err = exec.Command("./bpf/run_probes.sh", ".", ".").CombinedOutput()
-	if err != nil {
+	if err := exec.Command("./bpf/run_probes.sh", "./bpf", ".").Run(); err != nil {
 		log.Fatalf("BPF Verifier: NOT OK. Unable to run checker for bpf_features: %s", err)
 	}
-	bpfFeatures, err := ioutil.ReadFile(filepath.Join(globalsDir, "bpf_features.h"))
-	if err != nil {
+	if _, err := os.Stat(filepath.Join(globalsDir, "bpf_features.h")); os.IsNotExist(err) {
 		log.Fatalf("BPF Verifier: NOT OK. Unable to read bpf_features.h: %s", err)
 	}
-	if !strings.Contains(strings.ToLower(string(bpfFeatures)), "HAVE_MARK_MAP_VALS") {
```

Comment:
This warning is still valuable for us as it tells us whether the feature is there or not. I'm also fine with just dumping the contents of bpf_features.h in a compact format to the log.